### ISSUE TITLE
German Fdroid build

### ISSUE THREAD

**Issue by jugendhacker** [AnySoftKeyboard/LanguagePack#334]
I'm currently trying to get the German version updated on F-Droid, because I saw it was updated to the new source in Playstore. I only have two questions:
* Is there some kind of documentation how to build an specific language pack?
* Could you tag the version released on Playstore?

**Comment by martholomew** [comment]
I think there is something about F-Droid that isn't working, @menny do you know what's up?

**Comment by menny** [comment]
there is an issue with fdroid: https://gitlab.com/fdroid/fdroiddata/issues/1882#note_266142736

**Comment by menny** [comment]
regarding the technicals of how to publish to fdroid, in the upcoming weeks I'll add an artifact to the CI deploy task that will generate the required fdroid entries (this is similar to what's already [available](https://github.com/AnySoftKeyboard/AnySoftKeyboard/suites/396480217/artifacts/1079002) in AnySoftKeyboard).

**Comment by jugendhacker** [comment]
@menny you could also just tag the versions you release in playstore and fdroid could pick it up automatically.....

**Comment by menny** [comment]
I'm not aware of that mechanism. how do we set that up?

**Comment by jugendhacker** [comment]
You can find it [here](https://f-droid.org/en/docs/Build_Metadata_Reference/#UpdateCheckMode)... you need to set it to Tags

**Comment by menny** [comment]
I don't think it will work for us, due to the way we version the packs (if I understand correctly).
We have automatic versioning, where each commit, or build, produce a new version.
Our `AndroidManifest` file does not include version information, instead, it is calculated in build time.
